tests/unit/amo/reducers/test_collections.js

```diff
@@ -665,7 +667,15 @@ describe(__filename, () => {
     it('throws an error when addons are missing', () => {
       expect(() => {
         loadCurrentCollectionPage();
-      }).toThrow('addons are required');
+      }).toThrow('The addons parameter is required');
+    });
+
+    it('throws an error when numberOfAddons is missing', () => {
```

Comment:
The same goes for these two test cases: we can remove them if all they are testing is the `invariant` checks.